Issue1〜3の修正

genzisan opened the pull request:
よろしくお願いします
不都合やGITHUBの使い方が間違っていればアドバイス頂けると幸いです

genzisan committed:
lssu3

genzisan committed:
lssu1

genzisan committed:
Revert "lssu1"

This reverts commit 8c3180dbe187180923ea82be2c704a9856396c51.

genzisan committed:
コマンドテスト

genzisan committed:
lssu3

genzisan committed:
Revert "lssu3"

This reverts commit d0873898b5a1de5f3b273d220f961d32386e0db1.

genzisan committed:
lssu2

yupimaru commented on app/controllers/calendars_controller.rb:
```diff
@@ -18,7 +18,7 @@ def plan_params
   end
 
-  def getWeek
+  def get_Week
```
```ruby
def get_Week
```

こちら着眼点はとても素晴らしいですね。
メソッド名の命名規則として、**全て小文字のスネークケース**とありますので、
現在大文字になっている`W`は小文字の`w`にしておきましょう。

また、このメソッドを実際に呼び出している**5行目**の記述も同様に修正しましょう。

genzisan replied:
コード修正しました！
このレビューの返信方法で合ってるでしょうか？
お手数おかけしますが、よろしくお願いします。

yupimaru reviewed (commented):
レビュー依頼ありがとうございます。

Issue2,Issue3を解決するための修正はきちんと出来ていますね。

Issue1を解決するための修正が不足しているように見受けられます。
Issue1は、「古い記述であるハッシュロケットを使用してしまっている。これを、「シンボル型」に置き換えたい。」という内容でしたね。
**app/controllers/calendars_controller.rb**内に、以下のようなハッシュロケット記法を使用している箇所が残っていないか、再度確認してみましょう。
```ruby
# ハッシュロケット
{:sample => 'value'}　　

# シンボル型
{sample: 'value'}
```

genzisan committed:
lssu2の改善

genzisan committed:
lssu1の修正

1回目のプルリクエストでコードが届いていなかったでしょうか？
もしそうであれば申し訳ありません。

今回のlssu1ですが、キーとバリューでバリューに該当するものがなく、
今回全部シンボルを付けましたが、この記述で合ってあるでしょうか？

お忙しい中、コードの確認と修正以来くださりありがとうございます！

yupimaru commented on app/controllers/calendars_controller.rb:
```diff
@@ -2,7 +2,7 @@ class CalendarsController < ApplicationController
   # １週間のカレンダーと予定が表示されるページ
   def index
-    getWeek
+    get_Week
```
5行目
```ruby
get_Week
```

21行目の修正はきちんと出来ていますね。
21行目で定義したメソッドをこちらの5行目の記述で呼び出しているようです。

そのため、上記記述においても大文字の`W`は小文字の`w`に修正しておきましょう。

yupimaru commented on app/controllers/calendars_controller.rb:
```diff
@@ -18,8 +18,8 @@ def plan_params
-  def getWeek
-    wdays = ['(日)','(月)','(火)','(水)','(木)','(金)','(土)']
+  def get_week
+    wdays = [(日)：,(月)：,(火)：,(水)：,(木)：,(金)：,(土)：]
```
22行目
```ruby
wdays = [(日)：,(月)：,(火)：,(水)：,(木)：,(金)：,(土)：]
```

こちらはハッシュロケットからシンボル型へ変更する部分ではありませんので、
元の記述に戻しておきましょう。

**37行目**をよく見てみましょう。ハッシュロケットになっているところがありそうです。

genzisan replied:
コード修正しました！
お手数おかけしますが、ご確認よろしくお願いします

yupimaru reviewed (commented):
レビュー依頼ありがとうございます！
修正点確認させていただきました。
返信方法も問題ございません、ありがとうございます。

追加でコメントいたしましたので、ご確認お願いします！

genzisan committed:
lssu2の修正

genzisan committed:
lssu1の修正

見るところを間違えていました。
ご確認よろしくお願いします

yupimaru reviewed (commented):
レビュー依頼ありがとうございます！
無事全てのissueを解決することができましたね！
LGTMとさせていただきます！お疲れ様でした！

masterブランチへこちらのブランチをマージ後、
次の課題に取り組みましょう！応援しております！
https://master.tech-camp.in/curriculums/4681

genzisan commented:
色々とアドバイス頂きありがとうございました！
